### DISCUSSION TITLE
Conflict with laravel version

### DISCUSSION THREAD

**Discussion by Kenny201** [spatie/laravel-query-builder#723] [Q&A] [answered]
laravel: 8.82.0
Issue:"
  Problem 1
    - Root composer.json requires spatie/laravel-query-builder ^5.0 -> satisfiable by spatie/laravel-query-builder[5.0.0].
    - spatie/laravel-query-builder 5.0.0 requires illuminate/database ^9.0 -> found illuminate/database[v9.0.0-beta.1, ..., 9.x-dev] but these were not loaded, likely because it conflicts with another require."

**Comment by tsparksh** [comment, marked as answer]
Hi @Kenny201,
```
composer require spatie/laravel-query-builder:*
```
Works for me (laravel-query-builder ^4.0.2 and laravel ^8.54)

**Reply by Kenny201** [reply]
Hi, @tsparksh, Thank you very much. It works.

**Reply by ap1969** [reply]
Same here, after having the same problem.

**Comment by anand4262** [comment]
hey @tsparksh even after trying your solution I am unable to resolve the error. My laravel version is 8.75
